tests/components/netatmo/test_diagnostics.py

```diff
@@ -34,11 +35,9 @@ async def test_entry_diagnostics(hass, hass_client, config_entry):
         "data": {
             "auth_implementation": "cloud",
             "token": {
-                "access_token": "**REDACTED**",
-                "restricted_access_token": "**REDACTED**",
+                "access_token": REDACTED,
```

Comment:
You could do the same on the Renault tests (use the constant)... or I can adjust in a follow-up